### PR TITLE
Add haste

### PR DESCRIPTION
**Pull request by nicholaswilde**
Signed-off-by: Nicholas Wilde <ncwilde43@gmail.com>

**Description of the change** Add `haste` image

**Benefits**

New container image

**Possible drawbacks**

None

**Applicable issues**

- fixes #https://github.com/k8s-at-home/charts/issues/784

**Additional information**

N/A

### CODE REVIEW

File: apps/haste/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ENV APP_DIR="/app"
+USER root
+# hadolint ignore=DL3008,DL3015,SC2039,SC2086
+RUN \
+  echo "**** install packages ****" \
+  && \
+  apt-get update \
+  && \
+  apt-get install -y --no-install-recommends \
+    nodejs \
```
Comment (onedr0p) [inline apps/haste/Dockerfile:45]: Do you think there's any benefit to pinning this to a specific nodejs version? Usually we don't care too much about this but nodejs is usually breaking things in different major versions.

https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions

Comment (nicholaswilde) [reply apps/haste/Dockerfile:45]: Typically, I pin package versions but I noticed that kah doesn't and so I didn't. I can pin the version. Do you like to install from the binary source or is `apt` okay?

Comment (onedr0p) [reply apps/haste/Dockerfile:45]: apt is okay, using that link I posted above

File: static/index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
+
```
Comment (onedr0p) [inline static/index.html:1]: What is this file?

**Review by onedr0p — COMMENTED** [review]
What is this file?

File: static/index.html (hunk 2 above)
Comment (nicholaswilde) [reply static/index.html]: Bah. A static file generated by haste when running the image. I've removed it and added it to the .gitignore file.

**Pull request merged**